### PR TITLE
[ONNX FE] Support for ONNX ATen (embedding bag)

### PR DESCRIPTION
**Pull request by mitruska**
### Details:
Support for [ONNX ATen](https://github.com/onnx/onnx/blob/rel-1.2.3/docs/Operators.md#experimental-aten) `embedding_bag` operator by [EmbeddingBagPackedSum ](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/sparse/EmbeddingBagPackedSum_3.md) and [EmbeddingBagOffsetsSum](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/sparse/EmbeddingBagOffsetsSum_3.md) 
based on MO [ATenToEmbeddingBag.py](https://github.com/openvinotoolkit/openvino/blob/master/model-optimizer/extensions/front/ATenToEmbeddingBag.py). 

ATen was previously in onnx standard as experimental.
It enables models from the ticket.

### Tickets:
- 69870

### CODE REVIEW

File: ngraph/frontend/onnx/frontend/src/op/aten.hpp [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+namespace ngraph {
+namespace onnx_import {
+namespace op {
+namespace set_1 {
+
+OutputVector aten(const Node& node);
+
+}  // namespace set_1
+}  // namespace op
+
```
[the remark this answers is not in the document]
Comment (mitruska) [reply ngraph/frontend/onnx/frontend/src/op/aten.hpp]: Removed

File: ngraph/frontend/onnx/frontend/src/op/aten.cpp [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                     operator_name);
+
+    const int64_t mode = node.get_attribute_value<int64_t>("mode");
+    CHECK_VALID_NODE(node,
+                     mode == 0,
+                     "Unsupported mode, only `0` (sum) is supported as ATen embedding_bag `mode` attribute. Got: ",
+                     mode);
+    CHECK_VALID_NODE(node, inputs.size() >= 2, "Minimum 2 inputs are required. Got: ", inputs.size());
+
+    Output<ov::Node> embedding_bag;
```
[the remark this answers is not in the document]
Comment (mitruska) [reply ngraph/frontend/onnx/frontend/src/op/aten.cpp]: Moved to line :42

File: ngraph/frontend/onnx/frontend/src/op/aten.cpp [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+namespace ngraph {
+namespace onnx_import {
+namespace op {
+namespace set_1 {
+
+OutputVector aten(const Node& node) {
+    OutputVector inputs{node.get_ng_inputs()};
+
+    std::string operator_name = node.get_attribute_value<std::string>("operator", "");
```
[the remark this answers is not in the document]
Comment (mitruska) [reply ngraph/frontend/onnx/frontend/src/op/aten.cpp]: Done

**Review by t-jankowski — APPROVED** [review]

File: ngraph/frontend/onnx/frontend/src/op/aten.cpp [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+        // Need to expand embedding table with zeros (default values for empty bags)
+        const auto& emb_tbl_in = inputs[0];
+        const auto& indices_in = inputs[1];
+        const auto& offsets_in = inputs[2];
+        const auto& per_sample_weights_in = inputs[3];
+
+        const auto data_type = emb_tbl_in.get_element_type();
+        const auto ind_type = indices_in.get_element_type();
+
+        const auto zero_const = std::make_shared<default_opset::Constant>(ind_type, Shape{}, 0);
```
Comment (tomdol) [inline ngraph/frontend/onnx/frontend/src/op/aten.cpp:58]: Is it intentional that some of the scalars wrapped in a `Constant` have shape `{}` and others `{1}`?

**Review by tomdol — APPROVED** [review]

File: ngraph/frontend/onnx/frontend/src/op/aten.cpp (hunk 4 above)
Comment (mitruska) [reply ngraph/frontend/onnx/frontend/src/op/aten.cpp:58]: Actually in the current version there is no usage of `one_const` at all, so I've removed it.
Some ops were compatible only with scalar shape or 1D shape, but in this case it shouldn't make difference.

**Review by mbencer — APPROVED** [review]

**Pull request merged**